src/templates/podcast.js

```diff
@@ -83,6 +80,29 @@ const StyledPlayer = styled(AudioPlayer)`
     }
 `;
 
+const StyledBlogShareLinks = styled(BlogShareLinks)`
+    flex-direction: column;
+    align-items: center;
+    > :not(:first-child) {
+        margin-top: ${size.medium};
+    }
+    @media ${screenSize.upToLarge} {
+        display: inline-flex;
+        flex-direction: row;
+        > * {
+            margin-top: 0;
+            margin-left: ${size.mediumLarge};
+        }
+    }
+`;
+
+const StyledFooter = styled(ShareFooter)`
+    /* Target the copy link, although it is below some divs */
+    div > div > a {
```

Comment:
Like nesting in CSS it's good, but we can import the styled components and use nesting with them, I prefer this way, but you can omit this suggestion.